### PR TITLE
Reduce non-generated code in sonar-ws

### PR DESCRIPTION
**Pull request by bartfastiel**

### CODE REVIEW

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 1]
```diff
@@ -51,17 +54,23 @@
...
+  private final OrganizationService organizationsOld;
+  private final OrganizationsService organizations;
+  private final org.sonarqube.ws.client.permission.PermissionsService permissionsOld;
+  private final PermissionsService permissions;
+  private final org.sonarqube.ws.client.component.ComponentsService componentsOld;
+  private final ComponentsService components;
   private final FavoritesService favoritesService;
-  private final QualityProfilesService qualityProfilesService;
-  private final IssuesService issuesService;
+  private final QualityProfilesService qualityProfilesOld;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "QualityProfilesService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 2]
```diff
@@ -70,23 +79,30 @@
...
+    this.organizationsOld = new OrganizationService(wsConnector);
+    this.organizations = new OrganizationsService(wsConnector);
+    this.permissionsOld = new org.sonarqube.ws.client.permission.PermissionsService(wsConnector);
+    this.permissions = new PermissionsService(wsConnector);
+    this.componentsOld = new org.sonarqube.ws.client.component.ComponentsService(wsConnector);
+    this.components = new ComponentsService(wsConnector);
     this.favoritesService = new FavoritesService(wsConnector);
-    this.qualityProfilesService = new QualityProfilesService(wsConnector);
-    this.issuesService = new IssuesService(wsConnector);
+    this.qualityProfilesOld = new QualityProfilesService(wsConnector);
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "QualityProfilesService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 3]
```diff
@@ -126,13 +158,23 @@ public FavoritesService favorites() {
   }
 
   @Override
-  public QualityProfilesService qualityProfiles() {
-    return qualityProfilesService;
+  public QualityProfilesService qualityProfilesOld() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "QualityProfilesService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 4]
```diff
@@ -51,17 +54,23 @@
...
+  private final org.sonarqube.ws.client.permission.PermissionsService permissionsOld;
+  private final PermissionsService permissions;
+  private final org.sonarqube.ws.client.component.ComponentsService componentsOld;
+  private final ComponentsService components;
   private final FavoritesService favoritesService;
-  private final QualityProfilesService qualityProfilesService;
-  private final IssuesService issuesService;
+  private final QualityProfilesService qualityProfilesOld;
+  private final QualityprofilesService qualityprofiles;
+  private final org.sonarqube.ws.client.issue.IssuesService issuesOld;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "IssuesService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 5]
```diff
@@ -70,23 +79,30 @@
...
+    this.permissionsOld = new org.sonarqube.ws.client.permission.PermissionsService(wsConnector);
+    this.permissions = new PermissionsService(wsConnector);
+    this.componentsOld = new org.sonarqube.ws.client.component.ComponentsService(wsConnector);
+    this.components = new ComponentsService(wsConnector);
     this.favoritesService = new FavoritesService(wsConnector);
-    this.qualityProfilesService = new QualityProfilesService(wsConnector);
-    this.issuesService = new IssuesService(wsConnector);
+    this.qualityProfilesOld = new QualityProfilesService(wsConnector);
+    this.qualityprofiles = new QualityprofilesService(wsConnector);
+    this.issuesOld = new org.sonarqube.ws.client.issue.IssuesService(wsConnector);
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "IssuesService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 6]
```diff
@@ -51,17 +54,23 @@
...
+  private final QualityProfilesService qualityProfilesOld;
+  private final QualityprofilesService qualityprofiles;
+  private final org.sonarqube.ws.client.issue.IssuesService issuesOld;
+  private final IssuesService issues;
   private final UsersService usersService;
   private final UserGroupsService userGroupsService;
   private final UserTokensService userTokensService;
   private final QualitygatesService qualityGatesService;
-  private final MeasuresService measuresService;
+  private final org.sonarqube.ws.client.measure.MeasuresService measuresOld;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "MeasuresService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 7]
```diff
@@ -70,23 +79,30 @@
...
+    this.qualityProfilesOld = new QualityProfilesService(wsConnector);
+    this.qualityprofiles = new QualityprofilesService(wsConnector);
+    this.issuesOld = new org.sonarqube.ws.client.issue.IssuesService(wsConnector);
+    this.issues = new IssuesService(wsConnector);
     this.usersService = new UsersService(wsConnector);
     this.userGroupsService = new UserGroupsService(wsConnector);
     this.userTokensService = new UserTokensService(wsConnector);
     this.qualityGatesService = new QualitygatesService(wsConnector);
-    this.measuresService = new MeasuresService(wsConnector);
+    this.measuresOld = new org.sonarqube.ws.client.measure.MeasuresService(wsConnector);
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "MeasuresService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 8]
```diff
@@ -106,18 +123,33 @@ public WsConnector wsConnector() {
...
+    return organizationsOld;
+  }
+
+  @Override
+  public OrganizationsService organizations() {
     return organizations;
   }
 
+  @Override
+  public org.sonarqube.ws.client.permission.PermissionsService permissionsOld() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "PermissionsService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 9]
```diff
@@ -155,9 +197,14 @@ public QualitygatesService qualityGates() {
     return qualityGatesService;
   }
 
+  @Override
+  public org.sonarqube.ws.client.measure.MeasuresService measuresOld() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "MeasuresService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 10]
```diff
@@ -126,13 +158,23 @@ public FavoritesService favorites() {
...
+    return qualityProfilesOld;
+  }
+
+  @Override
+  public QualityprofilesService qualityProfiles() {
+    return qualityprofiles;
+  }
+
+  @Override
+  public org.sonarqube.ws.client.issue.IssuesService issuesOld() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "IssuesService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 11]
```diff
@@ -70,23 +79,30 @@
   private final SettingsService settingsService;
   private final RootsService rootsService;
   private final WebhooksService webhooksService;
-  private final ProjectAnalysisService projectAnalysisService;
+  private final ProjectAnalysisService projectAnalysisOld;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "ProjectAnalysisService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 12]
```diff
@@ -95,7 +111,8 @@
     this.settingsService = new SettingsService(wsConnector);
     this.rootsService = new RootsService(wsConnector);
     this.webhooksService = new WebhooksService(wsConnector);
-    this.projectAnalysisService = new ProjectAnalysisService(wsConnector);
+    this.projectAnalysisOld = new ProjectAnalysisService(wsConnector);
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "ProjectAnalysisService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 13]
```diff
@@ -106,18 +123,33 @@ public WsConnector wsConnector() {
...
+  }
+
   @Override
   public PermissionsService permissions() {
-    return this.permissionsService;
+    return permissions;
+  }
+
+  @Override
+  public org.sonarqube.ws.client.component.ComponentsService componentsOld() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "ComponentsService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 14]
```diff
@@ -51,17 +54,23 @@
 class DefaultWsClient implements WsClient {
 
   private final WsConnector wsConnector;
-  private final OrganizationService organizations;
-  private final PermissionsService permissionsService;
-  private final ComponentsService componentsService;
+  private final OrganizationService organizationsOld;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "OrganizationService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 15]
```diff
@@ -70,23 +79,30 @@
...
+  private final ProjectAnalysesService projectAnalyses;
   private final NotificationsService notificationsService;
   private final ProjectBranchesService projectBranchesService;
 
   DefaultWsClient(WsConnector wsConnector) {
     this.wsConnector = wsConnector;
-    this.organizations = new OrganizationService(wsConnector);
-    this.permissionsService = new PermissionsService(wsConnector);
-    this.componentsService = new ComponentsService(wsConnector);
+    this.organizationsOld = new OrganizationService(wsConnector);
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "OrganizationService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 16]
```diff
@@ -51,17 +54,23 @@
 class DefaultWsClient implements WsClient {
 
   private final WsConnector wsConnector;
-  private final OrganizationService organizations;
-  private final PermissionsService permissionsService;
-  private final ComponentsService componentsService;
+  private final OrganizationService organizationsOld;
+  private final OrganizationsService organizations;
+  private final org.sonarqube.ws.client.permission.PermissionsService permissionsOld;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "PermissionsService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 17]
```diff
@@ -70,23 +79,30 @@
...
   private final ProjectBranchesService projectBranchesService;
 
   DefaultWsClient(WsConnector wsConnector) {
     this.wsConnector = wsConnector;
-    this.organizations = new OrganizationService(wsConnector);
-    this.permissionsService = new PermissionsService(wsConnector);
-    this.componentsService = new ComponentsService(wsConnector);
+    this.organizationsOld = new OrganizationService(wsConnector);
+    this.organizations = new OrganizationsService(wsConnector);
+    this.permissionsOld = new org.sonarqube.ws.client.permission.PermissionsService(wsConnector);
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "PermissionsService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 18]
```diff
@@ -201,8 +248,13 @@ public WebhooksService webhooks() {
   }
 
   @Override
-  public ProjectAnalysisService projectAnalysis() {
-    return projectAnalysisService;
+  public ProjectAnalysisService projectAnalysisOld() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "ProjectAnalysisService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 19]
```diff
@@ -51,17 +54,23 @@
...
 
   private final WsConnector wsConnector;
-  private final OrganizationService organizations;
-  private final PermissionsService permissionsService;
-  private final ComponentsService componentsService;
+  private final OrganizationService organizationsOld;
+  private final OrganizationsService organizations;
+  private final org.sonarqube.ws.client.permission.PermissionsService permissionsOld;
+  private final PermissionsService permissions;
+  private final org.sonarqube.ws.client.component.ComponentsService componentsOld;
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "ComponentsService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 20]
```diff
@@ -70,23 +79,30 @@
...
   DefaultWsClient(WsConnector wsConnector) {
     this.wsConnector = wsConnector;
-    this.organizations = new OrganizationService(wsConnector);
-    this.permissionsService = new PermissionsService(wsConnector);
-    this.componentsService = new ComponentsService(wsConnector);
+    this.organizationsOld = new OrganizationService(wsConnector);
+    this.organizations = new OrganizationsService(wsConnector);
+    this.permissionsOld = new org.sonarqube.ws.client.permission.PermissionsService(wsConnector);
+    this.permissions = new PermissionsService(wsConnector);
+    this.componentsOld = new org.sonarqube.ws.client.component.ComponentsService(wsConnector);
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "ComponentsService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java [hunk 21]
```diff
@@ -106,18 +123,33 @@ public WsConnector wsConnector() {
   }
 
   @Override
-  public OrganizationService organizations() {
+  public OrganizationService organizationsOld() {
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java]: MINOR Remove this use of "OrganizationService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java [hunk 22]
```diff
@@ -61,19 +64,50 @@
...
+  @Deprecated
+  OrganizationService organizationsOld();
+
+  OrganizationsService organizations();
+
+  /**
+   * @deprecated since 7.0 use {@link #components()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.component.ComponentsService componentsOld();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java [hunk 23]
```diff
@@ -61,19 +64,50 @@
...
+
   IssuesService issues();
 
   NotificationsService notifications();
 
+  /**
+   * @deprecated since 7.0 use {@link #permissions()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.permission.PermissionsService permissionsOld();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java [hunk 24]
```diff
@@ -120,8 +160,12 @@
 
   /**
    * @since 6.3
+   * @deprecated since 7.0 use {@link #projectAnalyses()} instead
    */
-  ProjectAnalysisService projectAnalysis();
+  @Deprecated
+  ProjectAnalysisService projectAnalysisOld();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java [hunk 25]
```diff
@@ -61,19 +64,50 @@
  * @since 5.3
  */
 public interface WsClient {
-  OrganizationService organizations();
+
+  /**
+   * @deprecated since 7.0 use {@link #organizations()} instead
+   */
+  @Deprecated
+  OrganizationService organizationsOld();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java [hunk 26]
```diff
@@ -61,19 +64,50 @@
...
 
   ComponentsService components();
 
   FavoritesService favorites();
 
+  /**
+   * @deprecated since 7.0 use {@link #issues()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.issue.IssuesService issuesOld();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java [hunk 27]
```diff
@@ -61,19 +64,50 @@
...
+  org.sonarqube.ws.client.permission.PermissionsService permissionsOld();
+
   PermissionsService permissions();
 
-  QualityProfilesService qualityProfiles();
+  /**
+   * @deprecated since 7.0 use {@link #qualityProfiles()} instead
+   */
+  @Deprecated
+  QualityProfilesService qualityProfilesOld();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java [hunk 28]
```diff
@@ -83,6 +117,12 @@
 
   QualitygatesService qualityGates();
 
+  /**
+   * @deprecated since 7.0 use {@link #measures()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.measure.MeasuresService measuresOld();
```
Comment (SonarTech) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/OrganizationTester.java [hunk 29]
```diff
@@ -136,6 +136,6 @@ private void verifyMembersGroupMembership(String userLogin, @Nullable Organizati
   }
 
   public OrganizationService service() {
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/OrganizationTester.java]: MINOR Remove this use of "OrganizationService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/OrganizationTester.java [hunk 30]
```diff
@@ -136,6 +136,6 @@ private void verifyMembersGroupMembership(String userLogin, @Nullable Organizati
...
-    return session.wsClient().organizations();
+    return session.wsClient().organizationsOld();
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/OrganizationTester.java]: MINOR Remove this use of "organizationsOld"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java [hunk 31]
```diff
@@ -94,7 +94,7 @@ public final User generateAdministrator(Organizations.Organization organization,
   @SafeVarargs
   public final User generateAdministratorOnDefaultOrganization(Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().organizations().addMember(DEFAULT_ORGANIZATION_KEY, user.getLogin());
+    session.wsClient().organizationsOld().addMember(DEFAULT_ORGANIZATION_KEY, user.getLogin());
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java]: MINOR Remove this use of "organizationsOld"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java [hunk 32]
```diff
@@ -105,14 +105,14 @@ public final User generateAdministratorOnDefaultOrganization(Consumer<CreateRequ
...
-    session.wsClient().organizations().addMember(organization.getKey(), user.getLogin());
+    session.wsClient().organizationsOld().addMember(organization.getKey(), user.getLogin());
     return user;
   }
 
   @SafeVarargs
   public final User generateMemberOfDefaultOrganization(Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().organizations().addMember(DEFAULT_ORGANIZATION_KEY, user.getLogin());
+    session.wsClient().organizationsOld().addMember(DEFAULT_ORGANIZATION_KEY, user.getLogin());
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java]: MINOR Remove this use of "organizationsOld"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java [hunk 33]
```diff
@@ -83,7 +83,7 @@ public final User generateAdministrator(Consumer<CreateRequest.Builder>... popul
   public final User generateAdministrator(Organizations.Organization organization, Consumer<CreateRequest.Builder>... populators) {
     String organizationKey = organization.getKey();
     User user = generate(populators);
-    session.wsClient().organizations().addMember(organizationKey, user.getLogin());
+    session.wsClient().organizationsOld().addMember(organizationKey, user.getLogin());
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java]: MINOR Remove this use of "organizationsOld"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java [hunk 34]
```diff
@@ -105,14 +105,14 @@ public final User generateAdministratorOnDefaultOrganization(Consumer<CreateRequ
   @SafeVarargs
   public final User generateMember(Organizations.Organization organization, Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().organizations().addMember(organization.getKey(), user.getLogin());
+    session.wsClient().organizationsOld().addMember(organization.getKey(), user.getLogin());
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java]: MINOR Remove this use of "organizationsOld"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java [hunk 35]
```diff
@@ -74,7 +74,7 @@ public final User generate(Consumer<CreateRequest.Builder>... populators) {
   @SafeVarargs
   public final User generateAdministrator(Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().permissions().addUser(new org.sonarqube.ws.client.permission.AddUserRequest().setLogin(user.getLogin()).setPermission("admin"));
+    session.wsClient().permissionsOld().addUser(new org.sonarqube.ws.client.permission.AddUserRequest().setLogin(user.getLogin()).setPermission("admin"));
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java]: MINOR Remove this use of "permissionsOld"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/QProfileTester.java [hunk 36]
```diff
@@ -50,7 +50,7 @@
   }
 
   public QualityProfilesService service() {
```
Comment (SonarTech) [inline server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/QProfileTester.java]: MINOR Remove this use of "QualityProfilesService"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

**Comment by bartfastiel** [comment]
Closed in favor of #2864

**Pull request closed without merging**